syntax = "proto3";

package aggregate;

// import "google/protobuf/wrappers.proto";
import "protos/product.proto";

message ProductToAggregateRequest {
  product.Product product = 1;
}

message ProductToAggregateResponse {
  bool is_aggregated = 1;
}

service AggregationService {
  rpc AggregateProduct(ProductToAggregateRequest) returns (ProductToAggregateResponse);
}
